__test__/extensions/message-handlers/profanity-tagger.test.js

```diff
@@ -166,12 +178,15 @@ describe("Message Hanlder: profanity-tagger", () => {
     expect(user).toBe(true);
 
     // Confirm texter no-match
-    await sendMessage(c.testContacts[1].id, c.testTexterUser, {
-      userId: c.testTexterUser.id,
+    await sendMessage(c.testContacts[1].id.toString(), c.testTexterUser, {
```

Comment:
GREAT question!

I _think_ we're ok here. In the client code we're getting all the values via graphql interfaces and they will come in as strings. In the tests we're taking integers from the database (the IDs) and passing them to graphql queries and mutations. 

I don't think it's a good idea to change the type in the interface in order to avoid calling `toString()` and `parseInt()` in tests.

My reasoning when I made these changes was the UI code is internally consistent and the server code is internally consistent. 

However I'm keeping an open mind in case my assumptions fall apart when I start banging on the UI (which is now).